datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -720,5 +720,76 @@ CREATE TABLE dcsa_im_v3_0.vessel_schedule_terminal_visits (
     created_date_time timestamp with time zone NOT NULL DEFAULT now()
 );
 
+--- Supporting tables for Electronic Endorcement chain PoC
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.addressbookentry CASCADE;
+CREATE TABLE dcsa_im_v3_0.addressbookentry (

```

Comment:
The tables are inconsistent on whether they use `_` between words.  I think we should standardize to include `_` as that is what we have being doing everywhere else.

Reply:
fixed